lib/SendGridCompatibility/index.js

```diff
@@ -13,14 +14,15 @@ var SparkPost = require('../sparkpost');
  * @param apiKey: api key string
  * @param options: optional additional options object
  */
-var sendgrid = function(username, apiKey, options) {
+sendgrid = function(username, apiKey, options) {
+  var urlOpts, opts;
```

Comment:
usually we split vars on separate lines. this is fine for now, that would make a lot of updates for you for basically no meaningful impact :)